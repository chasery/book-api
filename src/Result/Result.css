.Result {
  display: flex;
  justify-content: center;
  list-style: none;
}
.Result + .Result {
  border-top: 1px solid lightgray;
}
.Result__image {
  margin-right: 1rem;
}
.Result__content {
  display: flex;
  flex-direction: column;
  text-align: center;
  font-size: 1.25rem;
}
.Result__details {
  list-style: none;
}
