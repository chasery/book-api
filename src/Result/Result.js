import React from "react";
import "./Result.css";

class Result extends React.Component {
  render() {
    return <div className="Result">Result</div>;
  }
}

export default Result;
