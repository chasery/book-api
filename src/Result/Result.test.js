import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Result from "./Result";

describe("Result component", () => {
  it("renders without error", () => {
    const wrapper = shallow(<Result />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
