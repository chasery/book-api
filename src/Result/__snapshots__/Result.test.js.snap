// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Result component renders without error 1`] = `
<div
  className="Result"
>
  Result
</div>
`;
